src/main/java/com/skhu/luxuryshop/product/dto/ProductRequestDto.java

```diff
@@ -0,0 +1,42 @@
+package com.skhu.luxuryshop.product.dto;
+
+import com.skhu.luxuryshop.product.entity.ProductEntity;
+import lombok.Builder;
+import lombok.Getter;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.PositiveOrZero;
+
+@Getter
+public class ProductRequestDto {
+    @NotBlank(message = "상품명이 없습니다")
+    private String name;
+    @NotBlank(message = "상품 설명이 없습니다")
+    private String content;
+    @PositiveOrZero(message = "숫자를 정확히 입력해주세요 ex) 500")
+    private Integer price;
+    private String category;
+    private String imageurl;
```

Comment:
카멜케이스로 하는게 맞지 않나요!??? 
```suggestion
    private String imageUrl;
```
그리고 그때 카톡으로 말했듯이 다중 이미지를 할 거라면 일급 컬렉션 객체로 포장해보아요.

Reply:
여기도 반영이 안돼있습니다~~~